qc/__init__.py

```diff
@@ -1,5 +1,8 @@
 # Main entry point for stdpopsim_qc
 
+from . qc import *  # NOQA
+
 # Species definitions.
 from . import homo_sapiens_qc  # NOQA
-from . import drosophlia_melanogaster_qc # NOQA
+from . import drosophlia_melanogaster_qc  # NOQA
+from . import arabidopsis_thaliana_qc  # NOQA
```

Comment:
do we need to also import Pongo here?

Reply:
Yep. But Pongo is currently disabled, and adding it here will fail some tests. If we get #363 merged first, I'll add it in. Otherwise the Pongo import will need to be added in #363.